### DISCUSSION TITLE
Browser doesn't works via 11.2.1.1 templates

### DISCUSSION THREAD

**Discussion by jeffersyuan1976** [AvaloniaUI/Avalonia#17603] [Q&A]
### Describe the bug

Hi,

I had updated the template via dotnet new update, the new version is 11.2.1.1.
And using dotnet new avalonia.xplat -o test to generate a new project.
Goto Browser folder, and dotnew run the new project
In browser, only show the logo picture. Developer method in console, it show the errors as following:

logging.ts:43 MONO_WASM: instantiate_wasm_module() failed TypeError: Cannot read properties of undefined (reading 'promise')
logging.ts:35 MONO_WASM: TypeError: Cannot read properties of undefined (reading 'promise')
    at http://localhost:5235/_framework/dotnet.runtime.o8gq1i8bk6.js:3:179338
startup.ts:500 Uncaught TypeError: Cannot read properties of undefined (reading 'promise')
    at startup.ts:500:71
5
startup.ts:500 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'promise')
    at startup.ts:500:71
assets.ts:136 Error in mono_download_assets: TypeError: Cannot read properties of undefined (reading 'promise')
startup.ts:500 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'promise')
    at startup.ts:500:71

![image](https://github.com/user-attachments/assets/529848d6-9dc2-4b85-ada1-1819e2d06822)

Any ideas on it?
Thanks.
Jeffers

### To Reproduce

As above.

### Avalonia version

11.2.1

### OS

WebAssembly

**Comment by maxkatz6** [comment]
1. Make sure you have latest .NET SDK patch release installed (either 8 or 9).
2. Make sure you have latest and up to date `wasm-tools` workload installed. In case if you are running "net8.0-browser" on .NET 9 SDK, you also need "wasm-tools-net8".
3. Clear bin/obj folder
4. Clear browser cache.

**Reply by MikeyWetzel** [reply]
I have the same problem as the OP. i upgraded to latest SDKs ("PS> dotnet --list-sdks" shows 8.0.405 and 9.0.102 as of 2/4/2025) and the problem remains the same. Exact same Console errors at OP.

I also upgraded the Avalonia nuget packages to their latest (11.2.3).
